openssl/src/ec.rs

```diff
@@ -459,12 +459,35 @@ impl EcKeyBuilderRef {
     pub fn generate_key(&mut self) -> Result<&mut EcKeyBuilderRef, ErrorStack> {
         unsafe { cvt(ffi::EC_KEY_generate_key(self.as_ptr())).map(|_| self) }
     }
+
+    /// Sets the public key based on affine coordinates.
+    pub fn set_public_key_affine_coordinates(&mut self,
+                                             x: &mut BigNumRef,
```

Comment:
Are these inputs modified by `EC_KEY_set_public_key_affine_coordinate`? The documentation doesn't appear to say one way or another.